### ISSUE TITLE
Support context menu in Files (UWP) app

### ISSUE THREAD

**Issue by pratyakshm** [M2Team/NanaZip#2]
Please try and check if its possible to add NanaZip to [Files UWP app](https://github.com/files-community/Files)'s context menu. Me (and hopefully) a lot of other people use it as our primary file manager app. Whilst Files does have its own unzipping mechanism, I would still believe that a third party integration like one with NanaZip would be robust and would benefit users.

**Comment by MouriNaruto** [comment]
![image](https://user-images.githubusercontent.com/10867563/134813846-028bf036-5cba-4aeb-a7bd-ebd8e929702e.png)
I have downloaded the Files UWP app just now and tested it. I found NanaZip's context menu supports perfectly in Files UWP app.

**Comment by emerysteele** [comment]
Confirmed working in the Files app. I had to scroll down in the context menu to get to it tho. Seems Files has a fixed context menu height.

**Comment by pratyakshm** [comment]
Oh right, I got it. I had to scroll down. Apparently Files app doesn't show the scrollbar in context menu by default, so I thought it isn't scroll-able. 😅


> 
> 
> Confirmed working in the Files app. I had to scroll down in the context menu to get to it tho. Seems Files has a fixed context menu height.

yeah, thanks man. In Files, when context menu is opened, scrollbar shall be showed for a few seconds before disappearing, it induces a sense of its presence. 

either way, thank you to both of you. closing this issue now! :D
